tests/test_reflect_class.py

```diff
@@ -0,0 +1,22 @@
+
+
+import unittest
+from jnius.reflect import autoclass, reflect_class
+
+class ReflectTest(unittest.TestCase):
+
+    def test_reflect_class(self):
+        cls_loader = autoclass("java.lang.ClassLoader").getSystemClassLoader()
+        # choose an obscure class that jnius hasnt seen before during unit tests
+        cls = cls_loader.loadClass("java.util.zip.CRC32")
```

Comment:
that looks a bit fragile, not sure what the best solution would be though, but maybe we could make sure it wasn't seen before, so the test would fail if another test had loaded it?

Reply:
you would have to expose jclass_register?